src/components/fields/DisplayExtendedField.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div v-if="showField" :class="'display-field-holder ' + columnClass + ' ' + (isChanged ? 'field-changed' : '')">
+    <div class="wrapper">
+      <div class="display-field-container">
+        <div class="radio-holder inline-boxed" v-for="(prop, idx) in properties" :key="idx">
+          <input type="radio" :id="'radio-' + prop + uuid" :name="'display-field-' + uuid" :value="prop" v-model="value">
+          <label :for="'radio-' + prop + uuid" data-toggle="tooltip" data-placement="bottom" :title="getTooltip(prop)">
+            <span :class="'check-icon check-display-' + prop"></span>
+          </label>
+        </div>        
+      </div>
+      <inherit-dot v-if="!isInheriting" @trigger-inherit="inheritValue" :move-left="true" :inheriting-from="inheritingFrom"></inherit-dot>
+    </div>
+  </div>
+</template>
+
+<script>
+import { state, getCurrentFieldValue, getFieldName,
+  saveFieldData, checkIsFieldChanged, checkLogic, sendCssToFrame } from '../../store'
+import InheritDot from '../UI/InheritDot'
+import displayProperties from '../../libs/display-properties'
+import { tooltips } from '../../libs/tooltips'
+import createClass from '../../libs/column-class'
+import bus from '../../libs/bus'
+
+export default {
+  data() {
+    return {
+      state,
+      value: getCurrentFieldValue(this.data.fieldConfig),
+      properties: displayProperties.extendedProperties,
+      isInheriting: this.checkInheritance(),
+      inheritingFrom: this.data.fieldConfig.inheritingFrom,
+      isChanged: checkIsFieldChanged(this.data.fieldConfig),
+      showField: typeof this.data.fieldConfig.showField !== 'undefined'
+        ? this.data.fieldConfig.showField
+        : true,
+      fromReset: false,
+      uuid: Fliplet.guid()
+    }
+  },
+  components: {
+    InheritDot
+  },
+  props: {
+    data: Object
+  },
+  watch: {
+    value(newVal, oldVal) {
+      if (newVal != oldVal && !this.fromReset) {
```

Comment:
```suggestion
      if (newVal !== oldVal && !this.fromReset) {
```

Reply:
There are plenty other instances in this PR

Reply:
Updated all `!=` and `==` to `!==` and `===` respectively.